src/components/Layout/Layout.tsx

```diff
@@ -112,11 +112,15 @@ function AnimatedSidebar({
 }
 
 function ContentArea() {
-  const { state } = usePresentationMode();
-  const isViewingPresentation = state.type === 'presentation';
+  const { state: presentationState } = usePresentationMode();
+  const isViewingPresentation = presentationState.type === 'presentation';
   const isViewingPresentationInEditMode =
-    isViewingPresentation && state.isEditMode;
-
+    isViewingPresentation && presentationState.isEditMode;
+  const isPresenting = presentationState.type === 'presenting';
+  const isCollaborationBarActive =
+    presentationState.type === 'idle' ||
+    isPresenting ||
+    (isViewingPresentation && presentationState.isEditMode);
```

Comment:
This is combination of expressions that are defined as well in `CollaborationBar` already. Why not to use the same expressions to have it much more explicit that `CollaborationBar` component is rendered when at least one of `ShowCollaboratorsCursorsToggle` or `Collaborators` is rendered? For these changes the tests are missing.

Reply:
this is for now a temporary check, the check will be removed in an upcoming ticket.

Reply:
see code update, thanks